nautobot/circuits/forms.py

```diff
@@ -223,7 +279,7 @@ class CircuitFilterForm(BootstrapMixin, TenancyFilterForm, StatusFilterFormMixin
         "q",
         "type",
         "provider",
-        "status",
+        "providernetwork" "status",
```

Comment:
typo?

Reply:
Unsure, ProviderNetwork is not part of circuits, therefore I removed it.